src/webui/src/App.tsx

```diff
@@ -177,8 +187,7 @@ class App extends React.Component<{}, AppState> {
         // experiment status and /trial-jobs api's status could decide website update
         if (['DONE', 'ERROR', 'STOPPED'].includes(EXPERIMENT.status) || TRIALS.jobListError()) {
             // experiment finished, refresh once more to ensure consistency
-            this.setState({ interval: 0 });
-            this.lastRefresh();
```

Comment:
I wrote `lastRefresh` because due to async nature when experiment status become `DONE`, some state may have not updated to their final value yet. For example maybe the last trial is still running at last `trial-jobs` REST call.